CHANGELOG.md

```diff
@@ -17,11 +17,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 ### Removed
 - insecure_data_transport
+- sensitive_data_exposure.mixed_content.sensitive_data_disclosure
+- sensitive_data_exposure.mixed_content.requires_being_a_man_in_the_middle
```

Comment:
In terms of ordering, I was initially thinking to keep it in order by when merged (always add to the bottom of each of the categories), thoughts?